### PR TITLE
Move the entry_points definitions into setup.py

### LINKED ISSUE #2294: Define entrypoints in setup.py instead of meta.yaml
Right now all entrypoints except for msui are declared in meta.yaml only. This has the disadvantage that they are bound to a single package definition for a single package manager. If we moved the entrypoints to setup.py any somewhat decent package manager would pick them up automatically (e.g. nix or pip). They could be dropped from the mss-feedstock as well then, I'd assume.

### PR DESCRIPTION
**Pull request by matrss**
Fixes #2294.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,6 +52,15 @@
     zip_safe=False,
     install_requires=[],  # we use conda build recipe
     entry_points=dict(
-        console_scripts=['msui = mslib.msui.msui:main'],
+        console_scripts=[
+            "mscolab = mslib.mscolab.mscolab:main",
+            "mswms = mslib.mswms.mswms:main",
+            "mswms_demodata = mslib.mswms.demodata:main",
+        ],
+        gui_scripts=[
```
Comment (ReimarBauer) [inline setup.py]: https://packaging.python.org/en/latest/specifications/entry-points/

The difference between console_scripts and gui_scripts only affects Windows systems. console_scripts are wrapped in a console executable, so they are attached to a console and can use [sys.stdin](https://docs.python.org/3/library/sys.html#sys.stdin), [sys.stdout](https://docs.python.org/3/library/sys.html#sys.stdout) and [sys.stderr](https://docs.python.org/3/library/sys.html#sys.stderr) for input and output. gui_scripts are wrapped in a GUI executable, so they can be started without a console, but cannot use standard streams unless application code redirects them. Other platforms do not have the same distinction.

We maybe should not do that extra for windows. Also we have some argparse arguments for msui  and click on mssautoplot.

There is also a windows problem which needs on some configurations a terminal start from msui.

Comment (ReimarBauer) [reply setup.py]: https://github.com/Open-MSS/MSS/issues/2261

Comment (matrss) [reply setup.py]: > We maybe should not do that extra for windows. Also we have some argparse arguments for msui and click on mssautoplot.

Commandline arguments are unrelated to stdin/stdout/stderr so AFAIU they should work perfectly fine with gui_scripts as well.

But I can move them all into console_scripts as well. EDIT: done.

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**